linux-tools/pexpect_test/pexpect_test.py

```diff
@@ -30,6 +30,10 @@ def run_once(self, test_path=''):
         """
         try:
             os.environ["LTPBIN"] = "%s/shared" %(test_path)
+            cwd = os.getcwd()
+            os.chdir("%s/pexpect_test" %(test_path))
+            os.system("patch -p0 < pexpect-pxssh-scripts.diff")
+            os.chdir(cwd)
```

Comment:
`cwd` is used only once, instead you can directly use it as,
`os.chdir(os.getcwd())`